From 82d684480b498445dd050efd2065252557108d41 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Mon, 10 Jul 2017 10:48:41 +0800
Subject: [PATCH 2097/5242] MLK-15932-1 video: dpu: Add prefix 'LB_' to member
 names of enum lb_mode_t

commit  156683ebbb2a944079130f6d64d8c0d747cbe9de from
https://source.codeaurora.org/external/imx/linux-imx.git

The member name 'NEUTRAL' of enum lb_mode_t is a little bit too generic,
since others DPU units have neutral modes as well, e.g., HScaler, VScaler,
Rop, CLuT and Matrix, etc.  So, let's add the prefix 'LB_' to member names
of enum lb_mode_t so that they can be specific to LayerBlends.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/dpu/dpu-plane.c |    2 +-
 include/video/dpu.h                 |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/imx/dpu/dpu-plane.c b/drivers/gpu/drm/imx/dpu/dpu-plane.c
index e3e1b60..f9c6bb7 100644
--- a/drivers/gpu/drm/imx/dpu/dpu-plane.c
+++ b/drivers/gpu/drm/imx/dpu/dpu-plane.c
@@ -249,7 +249,7 @@ static void dpu_plane_atomic_update(struct drm_plane *plane,
 
 	layerblend_pixengcfg_dynamic_prim_sel(lb, dpstate->stage);
 	layerblend_pixengcfg_dynamic_sec_sel(lb, dpstate->source);
-	layerblend_control(lb, BLEND);
+	layerblend_control(lb, LB_BLEND);
 	layerblend_pixengcfg_clken(lb, CLKEN__AUTOMATIC);
 	layerblend_position(lb, dpstate->layer_x, dpstate->layer_y);
 
diff --git a/include/video/dpu.h b/include/video/dpu.h
index 1400e67..4418cc6 100644
--- a/include/video/dpu.h
+++ b/include/video/dpu.h
@@ -340,8 +340,8 @@ enum dpu_irq {
 } lb_shadow_sel_t;
 
 typedef enum {
-	NEUTRAL,	/* Output is same as primary input. */
-	BLEND,
+	LB_NEUTRAL,	/* Output is same as primary input. */
+	LB_BLEND,
 } lb_mode_t;
 
 #define CLKEN_MASK		(0x3 << 24)
-- 
1.7.9.5

